config/schedule.rb

```diff
@@ -20,6 +20,6 @@
 # Learn more: http://github.com/javan/whenever
 
 # Run on production at 04:00 am EST or 3:00 am EDT
```

Comment:
Could you update this comment too, please?

Reply:
My head hurts! I think the comment is correct though. If the job runs at 9am UTC, that makes it 4am EST (or 3am EDT during daylight savings). I think?